docs/notebook-demos/ingesting-an-image.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ingesting an Image\n",
+    "\n",
+    "This example shows how to ingest a grayscale image and save the results locally. It is assumed that you have already read through the guides before beginning this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting the Data\n",
+    "\n",
+    "Before we can begin with the ingest, we must first download the data from S3. This curl command will download a file from S3 and save it to your `/tmp` direcotry.\n",
+    "\n",
+    "**A side note**: Files can be retrieved directly from S3 using the methods shown in this tutorial. However, this could not be done in this instance due to permission requirements needed to access the file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!curl -o /tmp/cropped.tif https://s3.amazonaws.com/geopyspark-test/example-files/cropped.tif"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Code\n",
+    "\n",
+    "With our file downloaded we can begin the ingest."
```

Comment:
this may be the case to explain more deeply what ingesting an image means - or link to it if that info is clearly explained elsewhere. aka. to answer the question - what is the difference between downloading something and ingesting something